stdlib/array.gr

```diff
@@ -552,6 +605,9 @@ provide let toList = array => {
  *
  * @param list: The list to convert
  * @returns The array containing all elements from the list
+ * 
+ * @example
+ * Array.fromList([1, 2, 3, 4]) // [1, 2, 3, 4] becomes [> 1, 2, 3, 4]
```

Comment:
same As Above:
```suggestion
 * Array.fromList([1, 2, 3, 4]) == [> 1, 2, 3, 4]
```

Reply:
See my comment on .toList.  Prefer the as submitted format for clarity and completness.